### DISCUSSION TITLE
Security Onion IDH Scan

### DISCUSSION THREAD

**Discussion by Tredmond7511** [Security-Onion-Solutions/securityonion#8936] [Unsupported Versions] [answered]
We are testing out the security onion honeypot feature ans so far it works great except for one issue. We try to scan it with Nmap and it displays that all 1000 ports are filtered. For our needs we need the honeypot to come up on network scans. I've looked through the docs and the configuration but haven't found much. I've opened all ports on iptables but no luck. I feel like I'm missing something simple but can't figure it out. Any help would be greatly appreciated.

**Comment by reyesj2** [comment]
How did you open the ports in iptables? Do the ports show when you run ```iptables -L -n```

**Reply by Tredmond7511** [reply]
I tried multiple specific allow rules for the installed services on the honeypot but nothing worked so I tried iptables -A INPUT -j ACCEPT. That didn't work so I flushed the rules with iptables -F then did the previously mentioned rule. That should have just opened all ports

**Reply by reyesj2** [reply]
Are you trying to have the IDH node respond over any port?
Do you have any service running on those ports or anything listening for incoming connections?

Also remember the host firewall is managed by salt.

**Reply by Tredmond7511** [reply]
I want the idh to respond to the network scans on the ports its emulating such as http and ftp. We can access the web server over http but network scans show the port as filtered which wouldn't lure our upcoming red team. I forgot about the salt firewall I will have to try that tomorrow.

**Reply by reyesj2** [reply]
If using nmap what command are you using to run your scan?

**Reply by Tredmond7511** [reply]
We've tried Nmap with the standard ip address,
Nmap -On, and Nmap -sT

**Comment by defensivedepth** [comment, marked as answer]
I just did a standard scan against an IDH node and it comes back as expected:

![image](https://user-images.githubusercontent.com/954732/196692882-fb0cb64f-3e8f-4737-bb46-6ae8b1a0ef43.png)


Couple things to note:

- You shouldn't need to manually configure the firewall at all - that is all taken care of for you. Just select the services you want to run and the firewall is all configured.
 
- Are you doing the nmap scan on the same network segment? Is it possible that there is some type of packet filter between you and the IDH node that is blocking it?

**Reply by Tredmond7511** [reply]
The first scan was external so I'm guessing our perimeter firewall caught that. However, the second scan was on the same subnet and switch LAN as the idh and it showed up filtered. The idh is running the services because I can interface with then on workstations. I'm using security onion .130 if that contributes anything

**Reply by defensivedepth** [reply]
From the system that you are running nmap from:

Can you access the IDH services? ie open http:// port 80 in the browser etc?

**Reply by Tredmond7511** [reply]
I did another install with just the basic web server option but all ports are still filtered

**Reply by defensivedepth** [reply]
Again, from the system that you are running nmap from - can you curl the basic web server:

![image](https://user-images.githubusercontent.com/954732/196726800-43be7bde-305e-4a4b-a0e9-16f119a404ff.png)

**Reply by Tredmond7511** [reply]
Figured out the issue, the version I was on, .130, was too early and I'm guessing didn't have the fu IDH functionality. Once I switched to .180 i could scan it no issue immediately

**Comment by Tredmond7511** [comment]
It's a pure cli that has no browser. I'm installing idh, I did custom port set up in the initial setup script maybe it has issue with that?
